src/App.css

```diff
@@ -85,33 +175,189 @@
   flex-grow: 1;
   text-align: left;
   margin-right: 12px;
+  width: 12.313rem;
+  height: 1.813rem;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: var(--color-black);
+  align-items: center;
 }
 
+/* form  */
+
 .form-wrapper {
   display: flex;
   flex-direction: row;
   align-items: center;
   margin-top: 20px;
-  margin-right: 92px;
+}
+
+.app-form {
+  display: flex;
+  flex: 1;
+  margin-bottom: 20px;
+}
+
+.app-form input {
+  width: 40%;
+  height: 2.5rem;
+  border-radius: 2px;
+  border: solid 1px var(--white-three);
+  background-color: var(--white-background-theme);
+  margin-right: 12px;
+}
+
+.app-form select {
+  width: 30%;
+  height: 2.7rem;
+  border-radius: 2px;
+  border: solid 1px var(--white-three);
+  background-color: var(--white-background-theme);
+  margin-right: 8px;
+}
+
+.add-btn {
+  flex: 1;
+  height: 2.5rem;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  margin-left: 12px;
+}
+
+.add-btn button {
+  width: 60%;
+  height: 2.063rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  border: none;
+  color: var(--white-background-theme);
```

Comment:
- [ ] Again here a reusable class would be the best way to use this style in our CSS as it is repeated 6 times. For more information refer to the first comment on how to apply it.